Psychtoolbox/PsychDemos/DrawFormattedText2Demo.m

```diff
@@ -0,0 +1,138 @@
+% DrawFormattedText2Demo
+%
+% Draws formatted text, shows how to align and center text vertically
+% and/or horizontally, how line-breaks are introduced, how to transform
+% text.
+%
+% Press any key to cycle through different demo displays.
+%
+% see also: PsychDemos, Screen DrawText?, DrawSomeTextDemo, DrawFormattedText2Demo
+
+% 09/15/16    dcn    Wrote it.
+
+try
+    AssertOpenGL;   % check PTB is functioning
```

Comment:
Ah, just see your reply. Thanks! Hmm, passing in 0.0-1.0... hexfloats are a bitch to work with. representing doubles as 16-long hexadecimal strings is unwieldy, so i guess i could simply parse some comma separated value here. And then its easy to support both. If there are commas in the value, it is 0.0-1.0, if not, it is 0-255. conversion of this will be done in either direction based on the colorrange of the window. Sounds complete?